en/docs/portal/reports.md

```diff
@@ -1,39 +1,16 @@
 # Reports
-
-HDT reports provide information about traffic to your HDT transports. Reports cover traffic volume, traffic bandwidth, and requests.
-
-If you are a reseller with child accounts, you can select the accounts that a report will cover (the parent account only, the child account only, or both the parent and child accounts).
-
-## Reports Page
-
-Reports are generated from the Reports page. To display this page, click **Reports** in the left pane.
-
-The following figure shows the key elements on the page.
+Clicking **Reports** in the left pane displays the Reports page.
 
 ![null](</docs/resources/images/reports/reports.png>)
 
-If you are a paid customer of **Virtual Private Line** usage, you can also switch between the “Total” and “Virtual Private Line” dashboard by clicking the selection button at the top right corner.
-
-## Understanding Report Types
-
-| **HDT Report Options**   | **Description**                               |
-| -------------------------|-----------------------------------------------|
-| Bandwidth                | Shows the total peak incoming and outgoing bandwidth, peak total bandwidth, average total bandwidth (in bps), and how the values change over time.|
-| Traffic                  | Shows the total amount of incoming and outgoing traffic, the total traffic value (in bytes), and how the values change over time.|
-| Requests                 | Shows the total number of requests and how the values change over time.|
-
-
-
+If you are a paid customer of Virtual Private Line usage, you can switch between the **Total** and **Virtual Private Line** dashboard by clicking the selection button at the top right corner.
 
 ## Generating Reports
-
-1. In the left pane, click **Reports**.
-2. Complete the fields in the Reports form. Required fields are denoted by an asterisk (\*).
-3. Click the **Generate Report** button to generate the report.
-4. With a generated report (similar to the one below) displayed at the bottom of the form, you can:
+1. Fill out the fields in the Reports form. Required fields are denoted by an asterisk (\*).
+2. Click the **Generate Report** button to generate the report.
 
 ![null](</docs/resources/images/reports/reports-generated.png>)
 
-  - Hold your cursor over data points in the report to view detailed information. The default time zone in the tooltips is from the local browser time. You can also overwrite it by changing the timezone in the "My Info".
-  - Use the <strong>Download</strong> button at the top right of the report to download the report in JPEG, PNG, and PDF formats.
+  - Hovering your cursor over the data points in a chart displays the detailed information. The default time zone in the tooltips is from the local browser time, you can overwrite it by changing the timezone in user info settings.
```

Comment:
don't separate sentences with a comma.
Use a period.

"The default time zone in the tooltips is from the local browser time.  You can overwrite it by ..."